### PR TITLE
Search for system_stm32fxxx.c locally before taking the template file…

### PR DESCRIPTION
**Pull request by alxhoff**
… from the CMSIS sources

### CODE REVIEW

File: cmake/FindCMSIS.cmake [hunk 1]
```diff
@@ -168,10 +168,19 @@ FOREACH(SRC ${CMSIS_DEVICE_SOURCES})
...
         CMAKE_FIND_ROOT_PATH_BOTH
     )
+    IF(NOT EXISTS ${CMSIS_${SRC_CLEAN}_FILE})
+        FIND_FILE(CMSIS_${SRC_CLEAN}_FILE ${SRC}
+            PATH_SUFFIXES src stm32${STM32_FAMILY_LOWER} cmsis
+            HINTS ${STM32Cube_DIR}/Drivers/CMSIS/Device/ST/STM32${STM32_FAMILY}xx/Source/Templates/
+            CMAKE_FIND_ROOT_PATH_BOTH
+        )
+    ELSE()
+        MESSAGE(STATUS "CMSIS system source found locally")
```
Comment (Hish15) [inline cmake/FindCMSIS.cmake:182]: Can you provide a message in all cases with the full path?

**Comment by atsju** [comment]
@alxhoff Could you please update your branch to solve the conflict ? 
We plan to merge as much PR as possible and we would appreciate your help if you want the modification to be merged.

**Comment by atsju** [comment]
duplicate of #137

**Comment by atsju** [comment]
#136 and #137 should be threated in 1 PR I suppose. I closed too fast thinking of duplicate

**Comment by atsju** [comment]
after looking at  #268 it seems the modern cmake approach of v2.0 release fulfills the need. This change is no longer needed.

**Pull request closed without merging**